slsim/Sources/SupernovaeCatalog/supernovae_sample.py

```diff
@@ -6,22 +6,52 @@
 import numpy as np
 from astropy import units
 from scipy import stats
-from astropy.coordinates import SkyCoord
 from slsim.Sources.galaxies import galaxy_projected_eccentricity
 from lenstronomy.Util.param_util import transform_e1e2_product_average
+from lenstronomy.Util.param_util import ellipticity2phi_q
 from slsim.Util.param_util import ellipticity_slsim_to_lenstronomy
 
 
+def elliptical_distortion_product_average(x, y, e1, e2, center_x, center_y):
```

Comment:
This function is more suitable in param util and we can keep using it even if this function will be included in lenstronomy.